libraries/AP_GPS/AP_GPS.h

```diff
@@ -242,6 +246,14 @@ class AP_GPS
         float relPosD;                     ///< Reported Vertical distance in meters
         float accHeading;                  ///< Reported Heading Accuracy in degrees
         uint32_t relposheading_ts;        ///< True if new data has been received since last time it was false
+#if GPS_SATELITES_INFO_ENABLED
+        uint8_t satellites_visible;
+        uint8_t satellites_used[20];
+        uint8_t satellites_elevation[20];
+        uint8_t satellites_azimuth[20];
+        uint8_t satellites_snr[20];
+        uint8_t satellites_prn[20];
```

Comment:
```suggestion
        struct {
            uint8_t visible;
            uint8_t used[20];
            uint8_t elevation[20];
            uint8_t azimuth[20];
            uint8_t snr[20];
            uint8_t prn[20];
        } satellites;
```

The constant 20 here is dubious; Needs to be a *lot* bigger than that.  Also needs to be a define.

We probably want to actually dynamically allocate the structure used to hold the sv if the user asks for the information to be streamed from the autopilot.